src/ofApp.h

```diff
@@ -3,9 +3,14 @@
 
 #include "ofMain.h"
 #include "ofxNetwork.h"
+#include "ofxOsc.h"
 
 #include "Test.h"
 
+#define OSC_PORT 7777
+#define COLOR_ADDRESS "/color"
```

Comment:
perhaps call it bgColor

Reply:
What do you think of `BG_COLOR_ADDRESS` since this is the OSC address for messages that change the background color (I believe `bgColor` is already a variable use to store the color value)

Reply:
Yeah